static/src/js/product_discount_field.js

```diff
@@ -0,0 +1,66 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+import { FloatField } from "@web/views/fields/float/float_field";
+import { _lt } from "@web/core/l10n/translation";
+import { ConfirmationDialog } from "@web/core/confirmation_dialog/confirmation_dialog";
+
+/**
+ * Dialog called if user changes a value in the sale order line.
+ * The wizard will open only if
+ *  (1) Sale order line is 3 or more
+ *  (2) First sale order line is changed
+ *  (3) value is the same in all other sale order line
```

Comment:
@iambenkis  can you change the mention of sale order line and write purchase order line?